src/EventSubscriber/GatedContentSubscriber.php

```diff
@@ -6,8 +6,8 @@
 use Drupal\Core\Cache\CacheableResponseInterface;
 use Drupal\Core\Session\AccountProxyInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
```

Comment:
Is it compatible with D8?

Reply:
Nope :disappointed: 

```
The website encountered an unexpected error. Please try again later.
TypeError: Argument 1 passed to Drupal\openy_gated_content\EventSubscriber\GatedContentSubscriber::accessCheck() must be an instance of Symfony\Component\HttpKernel\Event\RequestEvent, instance of Symfony\Component\HttpKernel\Event\GetResponseEvent given in Drupal\openy_gated_content\EventSubscriber\GatedContentSubscriber->accessCheck() (line 42 of /var/www/openy_gated_content/src/EventSubscriber/GatedContentSubscriber.php). 
```

Reverted this change. This because my local env on Drupal 9 with new symphony version